Changelog

```diff
@@ -164,10 +159,10 @@ Version 3.1, 2019-09-04
     * Do not increase fail counter when *checking* for an answered challenge (#1697)
     * Allow additional token specific checks when answering challenge response (#1695)
     * Endpoint GET /token/challenges also takes transaction_id (#1689)
-    * Push token can delay the response of /validate/check, so that there is no need 
```

Comment:
@laclaro Why did you change all the old text? This is annoying in a PR and makes it hard to read and has nothing to do with the issue. Can you please check if you have some nasty autoformatting?